recitation_1/grade (1).sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
can we rename this file to just grade.sh?

Reply:
Done !